src/test/java/com/wikia/webdriver/testcases/oasismainpage/CuratedContentTool.java

```diff
@@ -0,0 +1,28 @@
+package com.wikia.webdriver.testcases.oasismainpage;
+
+import com.wikia.webdriver.common.contentpatterns.PageContent;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.contentpatterns.PageContent'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)